setup.py

```diff
@@ -1,7 +1,12 @@
 """ Setup script for PyPI """
 import os
 from setuptools import setup
-from ConfigParser import SafeConfigParser
+
```

Comment:
Please remove this blank line and group the imports from the standard library together.